### PR TITLE
Add some metadata to some plugins

### PR DESCRIPTION
**Pull request by fabianfrz**
Quagga: dependency
SNMP: minimal package description

### CODE REVIEW

File: net-mgmt/snmp/pkg-descr [hunk 1]
```diff
@@ -0,0 +1,6 @@
+ bsnmp. It allows you to use SNMPv1 and SNMPv2 to query your OPNsense appliance
```
Comment (fichtner) [inline net-mgmt/snmp/pkg-descr]: Allow to use .... ~~your OPNsense appliance~~...

Possibly scrape the first bits of the manual page of the service or reuse the FreeBSD package descriptions altogether.

File: net-mgmt/snmp/pkg-descr [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+for information like the current interface status, CPU load etc.
+
+You will probably need this plugin if you want to use some sort of a monitoring tool.
```
Comment (fichtner) [inline net-mgmt/snmp/pkg-descr]: No guessing... just drop this

File: net-mgmt/snmp/pkg-descr [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+
+NOTE: This plugin is not using MVC. You cannot control it via the API.
```
Comment (fichtner) [inline net-mgmt/snmp/pkg-descr]: Not relevant to the user, but I think I get what you try to convey.

We should probably rather (a) provide a full API doc, or (b) provide an API doc per Plugin / Component in the GUI itself.

File: net/quagga/Makefile [hunk 4]
```diff
@@ -1,6 +1,7 @@
 PLUGIN_NAME=       quagga
 PLUGIN_VERSION=    0.0.1
 PLUGIN_COMMENT=    Quagga Routing Suite
+PLUGIN_DEPENDS=    quagga
```
Comment (fichtner) [inline net/quagga/Makefile]: good catch :D

File: net-mgmt/snmp/pkg-descr (hunk 1 above)
Comment (fabianfrz) [inline net-mgmt/snmp/pkg-descr]: The first bits of the man page are not really helpful:
```
The bsnmpd daemon serves the internet SNMP (Simple Network Management
     Protocol).  It is intended to serve only the absolute basic MIBs and
     implement all other MIBs through loadable modules.  In this way the
     bsnmpd can be used in unexpected ways.
```

Comment (fichtner) [inline net-mgmt/snmp/pkg-descr]: no, they are helpful in identifying the daemon is in itself not overly helpful :)

Comment (fichtner) [inline net-mgmt/snmp/pkg-descr]: and remember we have two mibs as modules in the ports tree that have pkg-descr

Comment (fabianfrz) [inline net-mgmt/snmp/pkg-descr]: those MIB modules are a dependency of this plugin.

Comment (fichtner) [inline net-mgmt/snmp/pkg-descr]: yes, exactly my point, they may have useful info in them that could be used as well

File: net/quagga/pkg-descr [hunk 5]
```diff
@@ -1,2 +1,9 @@
-Manage the Quagga Routing Software Suite, which offers routing protocols
-like RIP, IS-IS and OSPF to guide your packets through the sea.
+
```
Comment (fichtner) [inline net/quagga/pkg-descr]: empty line on top

File: net-mgmt/snmp/pkg-descr (hunk 1 above)
Comment (fabianfrz) [inline net-mgmt/snmp/pkg-descr]: https://github.com/opnsense/ports/blob/master/net-mgmt/bsnmp-regex/pkg-descr
https://github.com/opnsense/ports/blob/master/net-mgmt/bsnmp-ucd/pkg-descr

We would have to rewrite the descriptions but the main content could be used.

File: net/quagga/pkg-descr [hunk 6]
```diff
@@ -1,2 +1,9 @@
...
+Quagga is a routing software suite, providing implementations of
+OSPFv2, OSPFv3, RIP v1 and v2, RIPv3, BGPv4 and experemental ISIS
+for Unix platforms, particularly FreeBSD and Linux and also NetBSD,
+to mention a few. Quagga is a fork of GNU Zebra which was developed
+by KunihiroIshiguro. The Quagga tree aims to build a more involved
```
Comment (fichtner) [inline net/quagga/pkg-descr]: spacing for name is missing

Comment (fabianfrz) [inline net/quagga/pkg-descr]: this is 1:1 the same - an upstream problem ;)

Comment (fichtner) [inline net/quagga/pkg-descr]: you fix this one, I'll fix FreeBSD? :)

File: net-mgmt/snmp/pkg-descr [hunk 7]
```diff
@@ -0,0 +1,4 @@
+The snmp plugin allows you to query information about your OPNsense appliance about
+memory, load and CPU utilization and system statistics via the bsnmp-ucd module.
+It also allows to create counters from log files, program output and other text data via the bsnmp-regex module.
```
Comment (fabianfrz) [inline net-mgmt/snmp/pkg-descr]: this is more or less what the two dependencies have as an information a bit rewritten to fit better together. Still does not sound well, but it is better than nothing...

Comment (fichtner) [inline net-mgmt/snmp/pkg-descr]: I can tweak it a bit after merge if you want

Comment (fichtner) [inline net-mgmt/snmp/pkg-descr]: but sounds useful already

Comment (fabianfrz) [inline net-mgmt/snmp/pkg-descr]: @fichtner would appreciate it.

**Comment by fabianfrz** [comment]
@fichtner the description sounds better now.

**Comment by fabianfrz** [comment]
@fichtner Can I merge this one?

File: net/quagga/Makefile [hunk 8]
```diff
@@ -1,6 +1,7 @@
 PLUGIN_NAME=       quagga
 PLUGIN_VERSION=    0.0.1
```
Comment (fichtner) [inline net/quagga/Makefile]: This may conflict already. Otherwise go for it as discussed.

**Pull request merged**